### PR TITLE
Milestone 2: works and about section

### PR DESCRIPTION
**Pull request by dqarias**
In this milestone we:

-Build works and about section using best practices for HTML and CSS.
-Use Flexbox to place elements in the page.
-Use CSS Grid to place elements in the page.
-Use the template 1 that Microverse provide us.

### CODE REVIEW

File: css/about.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+.about__categories li{
+    height: 40px;
+    border-bottom: 1px solid #DFE1E6;
+    background-color: aquamarine;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+} */
```
Comment (Deepakdanger) [inline css/about.css]: - [ ] Kindly delete the commented code.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @dqarias, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/about.css (hunk 1 above)
Comment (dqarias) [reply css/about.css]: I've Just done it. Thank you for your feedback

**Comment by BrianSammit** [comment]
# STATUS: APPROVED :green_circle: :clap: 

Hello @dqarias
, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**